Dockerfile.dev

```diff
@@ -1,7 +1,7 @@
 FROM opensuse/tumbleweed
 
 RUN zypper --gpg-auto-import-keys refresh
-RUN zypper install -y openssh vim iputils ruby2.7 ruby2.7-devel ruby2.7-rubygem-cheetah ruby2.7-rubygem-nokogiri 
+RUN zypper install -y openssh vim iputils ruby3.1 ruby3.1-devel ruby3.1-rubygem-cheetah ruby3.1-rubygem-nokogiri
```

Comment:
I have to admit that I haven't tried this out... When developing, I simply spin up a Docker container with the official Ruby Docker image like this:

```
docker run -v "$PWD:/app" -it ruby:3.1.1 bash
```